fms/distributed/tensorparallel.py

```diff
@@ -44,6 +44,18 @@ def apply_embedding_tp(par_mod: nn.Embedding, mod: nn.Embedding, world_size, ran
     # print(f"For rank {rank}, we have the following weights: Base weight {mod.weight} bias {mod.bias}; Par weight {par_mod.weight}, bias {par_mod.bias}")
 
 
+def apply_moe_tp(par_mod: nn.Module, mod: nn.Module, param_names, world_size, rank):
+    with torch.no_grad():
+        for param in param_names:
+            par_param = getattr(par_mod, param)
+            print(param, par_param.shape[1] // world_size)
```

Comment:
did you intend to leave in the print statement?

Reply:
oops